src/app/pages/services/services.component.html

```diff
@@ -2,63 +2,61 @@
   <mat-card-content>
     <table ix-table [dataSource]="dataSource">
       <ng-container matColumnDef="name">
-        <th *matHeaderCellDef>{{ 'Name' | translate }}</th>
-        <td *matCellDef="let service; dataSource: dataSource">{{ serviceNames.get(service.service) }}</td>
+        <th mat-header-cell *matHeaderCellDef>{{ 'Name' | translate }}</th>
```

Comment:
I'm assuming we need those for tests. That's too bad :)

Reply:
Yeah, exactly. Otherwise, the table harness couldn't get values